metals/src/main/scala/scala/meta/metals/Semanticdbs.scala

```diff
@@ -84,8 +84,9 @@ object Semanticdbs extends LazyLogging {
     val bytes = Files.readAllBytes(semanticdbPath.toNIO)
     val sdb = Database.parseFrom(bytes)
     Database(
-      sdb.documents.map { d =>
-        val filename = s"file:${cwd.resolve(d.filename)}"
+      sdb.documents.map { d => //round and round we go
+        val absolute = cwd.resolve(d.filename)
+        val filename = absolute.toURI.toString
```

Comment:
Let's do `Uri(absolute).value` and then fix the broken implementation of `Uri.apply`

```
  def apply(uri: URI): Uri =
    new Uri(uri.toString) {}
```

This part here 

```
    if (uri.getScheme == "file") {
      // nio.Path.toUri.toString produces file:/// while LSP expected file:/
      new Uri(s"file://${uri.getPath}") {}
```
is broken since `URI.getPath` looses the URI encoding 🤦‍♂️